packages/codec/lib/ast/import/index.ts

```diff
@@ -476,10 +477,3 @@ export function definitionToStoredType(
     }
   }
 }
-
-//I am deliberately *NOT* exporting this.
-//If you have to make a type ID, instead make the type and then
-//take its ID.
```

Comment:
👍

Reply:
I mean, as you can see from the new comment, consumers of `codec` are still not supposed to use it -- the idea is that type IDs should be opaque and only constructed by import functions.  I'm just no longer enforcing that by not exporting it, because turns out more import functions than just these ones needed it.